Whiskey/Functions/New-WhiskeySemanticVersion.ps1

```diff
@@ -8,15 +8,28 @@ function New-WhiskeySemanticVersion
     .DESCRIPTION
     The `New-WhiskeySemanticVersion` function gets a semantic version that represents the current build. If called multiple times during a build, you'll get the same verson number back.
 
-    If passed a `Version`, it will return that version with build metadata attached. Any build metadata on the passed-in version is replaced. On a build server, build metadata is the build number, source control branch, and commit ID, e.g. `80.master.deadbee`. When run by developers, the build metadata is the current username and computer name, e.g. `whiskey.desktop001`.
-
-    Alternatively, you may pass a valid file path to the `Path` parameter and the function will pull the version number from that file.
+    If passed a `Version`, it will return that version with build metadata attached. Alternatively, you may pass a valid file path to the `Path` parameter and the function will pull the version number from that file. Any build metadata on the passed-in version is replaced. On a build server, build metadata is the build number, source control branch, and commit ID, e.g. `80.master.deadbee`. When run by developers, the build metadata is the current username and computer name, e.g. `whiskey.desktop001`.
 
     The `Path` parameter currently supports the following files:
 
-    * Node.js `package.json` files.
-    * PowerShell module manifest files with the file extension `.psd1`.
-    * .NET csproj files with the file extension `.csproj`.
+    * Node.js `package.json` files (JSON key: `version`).
+        ```

```

Comment:
Remove the three backticks. That is Github Markdown syntax. Indent by four spaces instead.

Reply:
Ah that's right. Fixed. Will need re-approval.